### ISSUE TITLE
street name fixes needed before import

### ISSUE THREAD

**Issue by vasony** [KAMI911/osm_poi_matchmaker#86]
- https://www.openstreetmap.org/way/299767035 
2040 Budaörs, Rubik utca -> Rubik Ernő utca

- https://www.openstreetmap.org/node/587836648
9436 Fertőszéplak, Petőfi utca -> Petőfi Sándor utca

- https://www.openstreetmap.org/node/2313016715
9431 Fertőd, Fő utca -> Fő út

- https://www.openstreetmap.org/node/2137197675
5650 Mezőberény, Kossuth Lajos tér -> Kossuth tér

- https://www.openstreetmap.org/node/3448565875
8284 Nemesgulács, Dózsa utca -> Dózsa György utca

- https://www.openstreetmap.org/node/8014479770 merged into https://www.openstreetmap.org/node/4454649803
5632 Békés, Szentpálsor utca -> Szent Pál sor

more to come

**Comment by KAMI911** [comment]
* OSM link: https://osm.org/node/6752850694 -->
addr:street                      M 'Fő út' 'Fő utca'
* OSM link: https://osm.org/node/5111696618 -->
addr:street                      M 'Rákóczi Ferenc tér' 'Rákóczi tér'
* OSM link: https://osm.org/node/1944057643 -->
addr:street                      M '56\-Osok tere' 'Ötvenhatosok tere'
* OSM link: https://osm.org/node/3487697464 -->
addr:street                      M 'Dr. Gréfin Lajos tér' 'Dr. Géfin Lajos tér'
* OSM link: https://osm.org/node/1806426701 -->

**Comment by vasony** [comment]
https://www.openstreetmap.org/node/371208501

Dorog, Mária út -> Mária utca

**Comment by vasony** [comment]
* duplicated space, e.g. "Zsirai Miklós  utca"
https://www.openstreetmap.org/node/8099990286

* extra information 
"Váralja (Pannonhalmi Főapátság turistafogadó)" , probably  simple "Vár" (the address of Pannonhalmi Főapátság is "Vár 1")
https://www.openstreetmap.org/node/8197146334

**Comment by vasony** [comment]
* Bányapart utca -> Bánya part
https://www.openstreetmap.org/node/8099990201

* Eszterházy J. utca -> Eszterházy János utca
https://www.openstreetmap.org/node/7972588472

* Zrinyi út -> Zrínyi Miklós út
https://www.openstreetmap.org/node/2294152592

**Comment by urbalazs** [comment]
- Nagytétényi út 27-47 -> Nagytétényi út 37-43 (forrás: https://campona.hu/elerhetoseg)
  https://www.openstreetmap.org/node/6752850719

**Comment by vasony** [comment]
* 1025 Budapest, Csatárka utca -> Csatárka út

https://www.openstreetmap.org/node/6178713322

**Comment by gy-mate** [comment]
@vasony I see that you created [`poi_patch.csv`](https://github.com/gy-mate/osm_poi_matchmaker/blob/a700901077316f3d95219ffb90b89f63a60c4d9d/osm_poi_matchmaker/data/poi_patch.csv) in 2f84bac. Is that table intended for fixing this issue? From what I see, it's not yet used by the app.

**Comment by vasony** [comment]
yes it is. but is is very old., so before we use it needs to verfiy( again ).the existence of the new way with the new name.

**Comment by gy-mate** [comment]
@vasony Is it okay for you if we do this manually until an automated solution is implemented?

I've queried all street names (together with the post code and city name) using Overpass and extended that table with a `new_street_exists_on_osm` column:

[`poi_patch_with_osm_address_exists.tsv`](https://github.com/user-attachments/files/26658267/poi_patch_with_osm_address_exists.tsv)

Could you please check all 240 rows that have a `FALSE` value in the new (last) column, update their `new_*` address values accordingly (if needed) and send back the corrected table? I'll open a PR for the changes.

**Comment by vasony** [comment]
yes but not now.  i'm out for a while for medical issues.

**Comment by gy-mate** [comment]
Sure, take your time and get well soon! :(

**Comment by vasony** [comment]
I started to fix the list, there are many valid steets in FALSE rows as well.
eg:
8308	Zalahaláp	Ady utca
3401	Mezőkövesd	Alkotmány út
3657	Királd	Alsótelep utca
9172	Győrzámoly	Ancsali út

whole list will be avaliable soon

**Comment by vasony** [comment]
I checked the FALSE rows, see the note column. Excel is full table, removed the original good addresses and I generated a new patch csv.

[address_patch.csv](https://github.com/user-attachments/files/28181019/address_patch.csv)
[address_patch.xlsx](https://github.com/user-attachments/files/28181020/address_patch.xlsx)

**Comment by gy-mate** [comment]
Awesome, thanks a lot! :)

[address_patch.csv](https://github.com/user-attachments/files/28181019/address_patch.csv) contains `orig_housenumber`s like `07.máj`. I guess Excel formatted the dashed house numbers as dates during the export.
Could you please fix them and regenerate the CSV?

**Comment by vasony** [comment]
Fixed 
[address_patch_20260524.csv](https://github.com/user-attachments/files/28187478/address_patch_20260524.csv)

**Comment by gy-mate** [comment]
Thank you! I've committed it in e54bcfe.
